cql3/statements/select_statement.cc

```diff
@@ -1927,6 +1927,21 @@ std::unique_ptr<prepared_statement> select_statement::prepare(data_dictionary::d
     // Force aggregation if GROUP BY is used. This will wrap every column x as first(x).
     if (!_group_by_columns.empty()) {
         aggregation_depth = std::max(aggregation_depth, 1u);
+        if (prepared_selectors.empty()) {
+            // We have a "SELECT * GROUP BY". If we leave prepared_selectors
+            // empty, below we choose selection::wildcard() for SELECT *, and
+            // forget to do the "levellize" trick needed for the GROUP BY.
```

Comment:
```suggestion
            // forget to do the "levelize" trick needed for the GROUP BY.
```

Reply:
Yes, but Avi used the spelling "levellize" in his original code, so I repeated it here (if you search the quoted word in the rest of this source file, you can see what this trick refers to).